fudge-ratio/src/main/webapp/index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
       <h1> Hello world!! Start files finished :-) </h1>
-      <p>Awesome</p>
+      <p>Awesome!</p>
```

Comment:
This first 2 commits are gold 😂

Reply:
Haha, I was checking if my changes were local to my machine.